super-test.sh

```diff
@@ -288,26 +288,27 @@ test "x$(which capnpc-c++)" = "x$STAGING/bin/capnpc-c++"
 
 cd samples
 
-doit capnp compile -oc++ addressbook.capnp -I"$STAGING"/include --no-standard-import
```

Comment:
I'd like to keep the existing test code that manually performs the compilation. Could we add the cmake-based build of the samples as an additional test (perhaps not part of the "quick" set)?

Reply:
Will do!